_posts/2020-05-08-april-2020.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: April 2020 Release Notes
+category: releases
+tags: release-notes
+---
+
+## Improvements
+
+* **Search.gov Website Migration:** Search.gov now uses the Federalist platform for its website.
+* **Analytics Changes:** Our team conducted a series of analytics upgrades to align with the forthcoming Elasticsearch upgrades.
+* **Rails Upgrade:** Search.gov now operates on Rails 5.2.4.2.
+* **Infrastructure Changes:** We increased the size of our infrastructure to accommodate a projected change in query volume due to the COVID-19 outbreak.
```

Comment:
Should this have been in the March release notes?

Reply:
Yes - I didn't put it in there since it was technically not a "release" but this month, thought it would be good to mention that since it was a good bit of our time. I changed this to "We maintained an upsized infrastructure to accommodate a projected change in query volume due to the COVID-19 outbreak."